helloworld-mutual-ssl-secured/configure-certs.cli

```diff
@@ -0,0 +1,19 @@
+#configure a key-store in the Elytron subsystem. The path to the keystore file doesn’t actually have to exist yet.
+/subsystem=elytron/key-store=mutualKS:add(path=clientCert.P12, relative-to=jboss.server.config.dir, credential-reference={clear-text=secret}, type=PKCS12)
```

Comment:
Maybe we should call it clientKS (just to make it more obvious that this is used for the client cert)

Reply:
The path could be client.keystore.p12 or something like that

Reply:
Thats true, I will change it to clientKS. 
WRT the path, I named it clientCert.p12 to make it consistent with the naming from the README file. In case someone wanted to use the cli script while going through the QS interactively. But I can change it too. Feel free to let me know what you think.

Reply:
Just checked the README. Looks like we are using "client.keystore" for the path there (see `keytool -genkey -keystore client.keystore ...`)